### DISCUSSION TITLE
No menu dock with browser/nautilus/other apps

### DISCUSSION THREAD

**Discussion by VeilSilence** [end-4/dots-hyprland#182] [Q&A] [answered]
Greetings. 
After power loss and unsafe turn off, some system files got corrupted and i can't manage to make dock with apps to appear.
I removed subvolumes "root" and "var" and keep only "home" as it was. Created new subvolumes, quickly reinstalled my system, log in and well... no dock. I'm sure it worked before power loss, so i'm out of ideas why it's not working at this point. Even tested on new user + fresh git pull and fresh install. Any ideas maybe? 
I will link ags error here just in case.
[debug.txt](https://github.com/end-4/dots-hyprland/files/13906735/debug.txt)

**Comment by end-4** [comment, marked as answer]
sometimes i comment out the dock when i feel like it because it serves no purpose in a tiling window manager
i don't find any real problem in the log
uncomment any line related to the dock in `~/.config/ags/config.js` if you want it

**Comment by VeilSilence** [comment]
I just want to verify that BTRFS file system not corrupted totally. 
Looks like everything works good now, thank you so much for assistance.
